### ISSUE TITLE
Windows Creators update - UWPInjector wont progress past "Remote Dumper Thread Found: 0x4590"

### ISSUE THREAD

**Issue by kishdr** [Wunkolo/UWPDumper#6]
![image](https://user-images.githubusercontent.com/17167456/31536303-16c2d32a-afcc-11e7-886f-442458f66709.png)

Ran UWPInjector, the blue text came up but it had an error copying a file for Halo Wars 2. Applied the patch BennettStaley wrote, only to get this problem. It never loads the blue, copy/downloading file text. My screen looked the same as on the OP's here - https://github.com/Wunkolo/UWPDumper/issues/5 

I thought this was just an error of me having an outdated version of UWPDumper-master so I reinstalled it.

Tried going through the Main.CCP to apply that patch BennettStaley wrote (https://github.com/Wunkolo/UWPDumper/issues/2) , only to find, I didn't have 

"fs::copy(
  		File.path(),
  		WritePath,
  		fs::copy_options::update_existing
  	)"

to begin with. UWPInjector still never loads without changing anything. I've tried relaunching the game itself and UWP, tried running UWP as admin... Nothing.

Not really sure whats going wrong. Any help is welcome.

**Comment by Wunkolo** [comment]
`Ran UWPInjector, the blue text came up but it had an error copying a file for Halo Wars 2.`
I recently had someone run into the same issue in which it would try to copy a file found within `data\textures` and would report an error which turned out to be because the file path and file name were massive(aka bigger than [MAX_PATH](https://msdn.microsoft.com/en-us/library/windows/desktop/aa365247(v=vs.85).aspx#maxpath)).
It would stop at the files found within `\data\textures\bespoke\archetypes\refuga_halo\rocks\tileable_skirts\...`.
Is this similar to your situation?
Are you running a release build with administrator rights as well?

**Comment by kishdr** [comment]
To Wunkolo,

I am running full administrator rights on the computer. Tried unlocking the HW2 files by giving permissions to myself but, didn't work. Not sure if it makes a difference, but eh.

Unfortunately I do not have any images of when UWP Dumper was errored but it was the first file it tried . Maybe it was the exe? I don't recall what the exact file was since, I haven't been able to see it in 12+ hours

**Comment by Wunkolo** [comment]
I recently pushed a bunch of changes and refactored some of the code that might fix your problem. Try the latest appveyor build and lemme know how it goes. Be sure to delete your current `DUMP` folder so that you start with a fresh and empty `TempState` folder. I also made the dumper automatically open explorer with the dump folder so you don't have to manually browse to it anymore.

**Comment by kishdr** [comment]
To Wunkolo,

I downloaded the new changes to the code and deleted the DUMP folder.

So far I am still not being able to get to the 'blue text'. I tried launching UWPInjector as admin, launching it regularly, as well as trying it with different UWP Apps, like Calculator. Same issue. The console remains "stuck" at Remote Dumper thread found: 0x3208. Should I have installed a newer UWPInjecter? 
![image](https://user-images.githubusercontent.com/17167456/31568146-dad7d8a4-b040-11e7-8962-45233d28d912.png)
